### ISSUE TITLE
Crash when entering the nether

### ISSUE THREAD

**Issue by DatrixTHLK** [CyclopsMC/EverlastingAbilities#24]
Crash Report: https://gist.github.com/DatrixTHLK/11a39704a03ebf39817bcb5c04c802e2

Minecraft: 1.10.2
Forge: 2125
Everlasting Abilities: 1.2.1

All I did was make a nether portal, then when I went in I spawned in the nether and crashed to desktop.

**Comment by rubensworks** [comment]
It doesn't look like this crash is caused by EA, it seems to be a Forge/Vanilla bug.

**Comment by DatrixTHLK** [comment]
I was hoping not. Going to the nether causes the crash and going back to the overworld causes a crash. Has someone mentioned this over at forge issues page yet?

**Comment by DatrixTHLK** [comment]
I mentioned it to forge [#3310](https://github.com/MinecraftForge/MinecraftForge/issues/3310)

**Comment by DatrixTHLK** [comment]
It's not with forge I can't reproduce this in a newly generated world both with and without mods. In the currently used world, however, it will crash when entering and exiting a nether portal. It says something about potion effect in the crash log. It doesn't mention any specific mod when I looked in the crash log though. forge 2107. This has also been mentioned to the howling moon mod author also. It could be howling moon, however, I have yet to hear back. 

There is one last check to determine that it could more likely be howling moon itself. My character is atm is a werewolf. I will turn back to normal and see if that will cause my crash going through nether portals. It's neither this mod, howling moon, nor Armour Reborn. I believe my current world is broken.

**Comment by DatrixTHLK** [comment]
If it's not your mod  and it is a forge Vanilla bug, why would my game crash when I get a certain ability, don't exactly know what ability that is though, as whatever happens crashes the game within the nether only. This hasn't happened until about 80-100 days into going to the nether. I thought this was fixed by now. Forge is not likely going to fix it. They don't answer my questions without giving me some remark about my mod list which I have tested by the way. I don't feel like asking them. I have loaded up an old save and the nether is fine until I get an ability of some kind. I will do some more testing on that then and let you know...

**Comment by rubensworks** [comment]
The only thing I can think of is that you are using a mod that is for some reason setting certain standard potion effects to `null`. I've just added a check which will disable abilities with null potion effects and a warning will be logged in this case.

You can try the latest dev build here: https://dl.bintray.com/cyclopsmc/dev/org/cyclops/everlastingabilities/EverlastingAbilities/1.9.4-1.2.1-37/EverlastingAbilities-1.9.4-1.2.1-37.jar

**Comment by DatrixTHLK** [comment]
I will try that then and see what happens. Thanks.

**EDIT:** Whatever happened before is still happening. The nether is still not working. I will take the mod out and see what happens. It is either this mod or Howling moon as these are the only two mods I can think of that may be conflicting with one another since both use the same abilities. As I said before, I will let you know what happens with this test.

**EDIT 2:** Took out your mod and it wasn't yours. Nether is still a red screen. Howling Moon mod was taken out and it wasn't that one either. 

**EDIT 3:** I'm going to just close this issue since I have no clue what is happening. Thanks for the help in the meantime.

**Comment by DatrixTHLK** [comment]
What would happen if you changed the max level of the abilities in the config file to something higher than what the abilities max were? Would that cause an error to occur?

For instance if the max level of Resistance is 5 and we set it to 2x that amount would that still gives us a max of 5 or would that give us a resistance of 10?

**Comment by rubensworks** [comment]
Not sure what you mean with that question.
But changing the max level in the config file, will simply change the max level of your ability, it's as simple as that.

**Comment by DatrixTHLK** [comment]
That's what I meant. Thanks anyways. I'm still trying to figure out what mod would change certain poition effects to null...

**Comment by DatrixTHLK** [comment]
I think this might be a solution. The potion effect nausea was set to 0 nullifying its effect and causing the crash going through a nether portal from nether to overworld. I'm not exactly sure how it got to be a level of simply 0. I'm hoping this is the cause all along, going to test it out. I could always be wrong...

> The maximum ability level.
>     I:nausea.maxLevel=0

That nausea potion effect happens whenever someone eats a Pufferfish or goes through a nether portal.  SInce being zero, my guess would be that, the game doesn't know how to handle that and just simply crashes. Then again I could always be wrong...

**Comment by rubensworks** [comment]
I think I correctly handle this case in the code, but let me know what the outcome of your test is.

**Comment by DatrixTHLK** [comment]
So far nothing out of the ordinary. The crash usually occurs within 80-100 minecraft days. Since I don't play minecraft 24/7, it will be at least one or two weeks before any results show.

**EDIT 10:57pm:** I've reached 133 days in minecraft and nothing has happened yet. I didn't think it would go as fast as it did. I might have miscalculated the days it takes or depends on how much activity I do in minecraft within those 100 days. The only mod I took out was armour reborn, that may have been the culprit all along. Too early to tell though.
